docrawl/docrawl_core.py

```diff
@@ -677,42 +697,10 @@ def _get_current_url(self, inp):
             :param page: :param browser: driver instance
             :param inp: list, inputs from launcher (filename)
         """
-
         filename = inp['filename']
         url = str(self.browser.current_url)
-
-        try:
-            with open(filename, 'w+', encoding="utf-8") as f:
-                f.write(url)
-        except Exception as e:
-            docrawl_logger.error(f'Error while getting current URL: {e}')
-
-    def _close_browser(self, inp):
-        """
-        Closes browser (removes driver instance).
-            :param browser: driver instance
-        """
-
-        try:
-            self.browser.quit()
-            '''
-            if not browser.current_url:
-                time.sleep(1)
-                close_browser(browser)
-            '''
-
-
-            # Remove proxy after closing browser instance
-            proxy = {'ip': '', 'port': '', 'username': '', 'password': ''}
-            browser_meta_data = self.docrawl_client.get_browser_meta_data()
-            browser_meta_data['browser']['proxy'] = proxy
```

Comment:
where did this part disappear?

Reply:
Test docrawl using proxies -> here you disabled it but it's pretty important for production